src/config/dynamic-theme-fixes.config

```diff
@@ -110,6 +110,13 @@ IGNORE INLINE STYLE
 
 ================================
 
+*.aliexpress.com
```

Comment:
Aliexpress already has a [config](https://github.com/darkreader/darkreader/blob/main/src/config/dynamic-theme-fixes.config#L1411-L1413), why you create a new config?

Reply:
Sorry, I didn't notice, I have updated the pull request.